Dockerfile

```diff
@@ -128,7 +128,7 @@ RUN /opt/ort/bin/import_proxy_certs.sh && \
     curl -ksS https://storage.googleapis.com/git-repo-downloads/repo > /usr/local/bin/repo && \
     chmod a+x /usr/local/bin/repo && \
     # Install package managers (in versions known to work).
-    npm install --global npm@$NPM_VERSION bower@$BOWER_VERSION yarn@$YARN_VERSION && \
+    npm install --location=global npm@$NPM_VERSION bower@$BOWER_VERSION yarn@$YARN_VERSION && \
```

Comment:
Commit message nits:
- "docker build" -> either "a Docker build" or "`docker build`"
- Typo in "deprecaeted".

Reply:
Commit message modified.